test/shared/utils.py

```diff
@@ -42,25 +42,39 @@ def cibuildwheel_run(project_path, env=None, add_env=None):
     )
 
 
-def expected_wheels(package_name, package_version):
+def expected_wheels(package_name, package_version, manylinux_versions={'1_x86_64', '2010_x86_64'}):
```

Comment:
This `manylinux_versions` param... why is the default x86_64 only? I'd say the default would be to expect all wheels. 

I'm wondering if perhaps a nicer signature would be like-
`def expected_wheels(package_name, package_version, i686=True, x86_64=True, manylinux1=True)`

Then tests can deselect the parts they don't need. (Just a suggestion!)

Reply:
Interesting, myeah. I wasn't fully happy with the signature either, but:

> why is the default x86_64 only?

Because currently there is only a `manylinux2010_i686` standard, but not an image :-( See [this reply](https://github.com/pypa/manylinux/issues/179#issuecomment-520523082). So, since the general consensus seems to be to move away from `manylinux1`, I made `manylinux2010_x86_64` the default image for 64-bit, and *nothing* the default image for 32-bit.
Happy to change opinion, though, if you disagree. Or maybe we should make a PR with `manylinux2010_i686` to make our code cleaner? :-P

> I'm wondering if perhaps a nicer signature would be like-
`def expected_wheels(package_name, package_version, i686=True, x86_64=True, manylinux1=True)`

Problem is that I was trying to be future-proof as well. A PEP on `manylinux2014` has already been approved, I believe, so I was hoping to limit the places where we'd need to copy-paste-replace2010by2014. But your signature is nicer, so I'd like to find a way in between!